style/water.mss

```diff
@@ -181,16 +183,30 @@
           background/line-width: 3;
         }
       }
+
+      [waterway = 'fish_pass'][zoom >= 15] {
```

Comment:
Can you merge this section with the one above for `[waterway = 'stream'][zoom >= 15]`?